tests/integration/test_remoted/test_active_response/test_active_response_send_ar.py

```diff
@@ -0,0 +1,93 @@
+    {'PROTOCOL': TCP, 'PORT': '1514'},
+    {'PROTOCOL': UDP, 'PORT': '1514'},
+    {'PROTOCOL': TCP_UDP, 'PORT': '1514'}
+
+]
+metadata = [
+    {'protocol': TCP, 'port': '1514'},
+    {'protocol': UDP, 'port': '1514'},
+    {'protocol': TCP_UDP, 'port': '1514'}
+]
+
+configurations = load_wazuh_configurations(configurations_path, __name__ ,
+                                           params=parameters, metadata=metadata)
+configuration_ids = [f"{x['PROTOCOL']}_{x['PORT']}" for x in parameters]
+
+manager_address = 'localhost'
+
+# fixtures
+@pytest.fixture(scope='module', params=configurations, ids=configuration_ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+def test_active_response_ar_sending(get_configuration, configure_environment, restart_remoted):
+    """Test if `wazuh-remoted` sends active response commands.
+
+    Check if execd sends active response command to remoted module and agent receives an active command
+    message from the manager.
+
+    Raises:
+        AssertionError: if `wazuh-remoted` does not send active response command or some of debug messages.
+    """
+    protocol_array = (get_configuration['metadata']['protocol']).split(',')
+    for protocol in protocol_array:
+        # rcv_msg_limit of 1000 is necessary for UDP test
+        agent = ag.Agent(manager_address, 'aes', os='debian8', version='4.2.0',
+                         disable_all_modules=True, rcv_msg_limit=1000)
+        agent.set_module_status('receive_messages', 'enabled')
+        agent.set_module_status('keepalive', 'enabled')
+
+        # Time necessary until socket creation
+        time.sleep(1)
+
+        sender = ag.Sender(manager_address, protocol=protocol)
+
+        injector = ag.Injector(sender, agent)
+
+        try:
+            injector.run()
+            agent.wait_status_active()
+
+            active_response_message = f"(local_source) [] NRN {agent.id} dummy-ar admin 1.1.1.1 1.1 44 (any-agent) " \
+                         "any->/testing/testing.txt - -"
+
+            send_active_response_message(active_response_message)
+
+            log_callback = remote.callback_active_response_received(active_response_message)
+            wazuh_log_monitor.start(timeout=5, callback=log_callback,
+                                    error_message='The expected event has not been found in ossec.log')
+
+            log_callback = remote.callback_active_response_sent(active_response_message)
+
+            wazuh_log_monitor.start(timeout=5, callback=log_callback,
+                                    error_message='The expected event has not been found in ossec.log')
+
+            remote.check_agent_received_message(agent.rcv_msg_queue, f'#!-execd dummy-ar admin 1.1.1.1 1.1 44 '
+                                                                     f'(any-agent) any->/testing/testing.txt - -',
+                                                                     escape=True)
```

Comment:
Add this AR message to the `remote` module as a variable.

Reply:
Done in 4e24efb4f8dcf8f8c05b75fe08b5820fb4f3305b,3cc7447ababd79cb7c49087dde0a2fbfe5cec5e1